stage1/app-rm/app-rm.go

```diff
@@ -18,31 +18,41 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"syscall"
 
 	"github.com/coreos/rkt/common"
 	rktlog "github.com/coreos/rkt/pkg/log"
 	stage1common "github.com/coreos/rkt/stage1/common"
 	stage1initcommon "github.com/coreos/rkt/stage1/init/common"
 
 	"github.com/appc/spec/schema/types"
+	"github.com/coreos/rkt/pkg/mountinfo"
 )
 
 var (
-	flagApp string
-	debug   bool
-	log     *rktlog.Logger
-	diag    *rktlog.Logger
+	debug     bool
+	flagApp   string
+	flagPhase int
+
+	log  *rktlog.Logger
+	diag *rktlog.Logger
 )
 
 func init() {
 	flag.StringVar(&flagApp, "app", "", "Application name")
 	flag.BoolVar(&debug, "debug", false, "Run in debug mode")
+
+	// `--phase` is not part of stage1 contract
+	flag.IntVar(&flagPhase, "phase", 0, "Removal phase, defaults to 0 when called from the outside")
```

Comment:
"phase 0" and "phase 1" are very abstract. We have established and documented terms for stages, where "stage 0" happens on the host, and "stage 1" happens in the context of the pod.

Can we either make the context of phases more explicit in the documentation, or even call this flag `stage`?